src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -75,7 +75,7 @@ def cluster_log(self, msg, lvl=None):
             lvl = self.mgr.CLUSTER_LOG_PRIO_WARN
         self.mgr.cluster_log("cluster", lvl, msg)
 
-    def volume_exception_to_retval(self, ve):
+    def exception_to_retval(self, ve):
```

Comment:
why change this?

Reply:
I was using the same routine for both VolumeException and CephfsConnectionException, so thought the name makes it generic ?

Reply:
Ummm.. ok. Does handling `CephfsConnectionException` in open_volume() makes sense?

Reply:
Makes sense, I have handled the exception in 'open_volume' and 'open_volume_lockless'